To prevent the below test failure:
Bail out! GLib-FATAL-WARNING: GError set over the top of a previous GError or uninitialized memory.
This indicates a bug in someone's code. You must ensure an error is NULL before it's set.
The overwriting error message was: Failed to find IP of interface lo

This is a crude hack, not suitable for upstream.
Also the tests do not pass yet due to other reasons (GnuTLS)
diff -r b9016cfda83c tests/gtest/gupnp-simple-igd.c
--- a/tests/gtest/gupnp-simple-igd.c	Wed Nov 04 17:47:38 2020 +0000
+++ b/tests/gtest/gupnp-simple-igd.c	Wed Nov 04 17:49:00 2020 +0000
@@ -258,7 +258,7 @@
     gpointer user_data)
 {
   if (!g_strcmp0 (gssdp_client_get_interface (GSSDP_CLIENT (gupnp_context)),
-          "lo"))
+          "lo0"))
     return FALSE;
   else
     return TRUE;
@@ -279,7 +279,7 @@
 
   if (mainctx)
     g_main_context_push_thread_default (mainctx);
-  context = gupnp_context_new (NULL, "lo", 0, NULL);
+  context = gupnp_context_new (NULL, "lo0", 0, NULL);
   g_assert (context);
 
   if (g_getenv ("XML_PATH"))
